libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -600,13 +600,24 @@ namespace SohImGui {
         ImGui::Text("%s", text);
     }
 
-    void EnhancementCheckbox(const char* text, const char* cvarName)
+    void EnhancementCheckbox(const char* text, const char* cvarName, bool disableCheckbox, const char* disableText)
```

Comment:
minor naming nitpick, i think just a bool `disabled` and then `disabledHoverText` or `disabledTooltipText` or something for the second would be nice

right now it seems like the first var disables the checkbox and the second disables the text (i know that doesn't make sense with the type on `disableText` but it'd still be nice for this to be as clear as possible since we'll be using it all over the place)

Reply:
That's fair. I'll go for `disabled` and `disabledToolTipText`.